bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/CommonInterface.java

```diff
@@ -221,15 +226,14 @@ default void commonInitialize() {
             setThingStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED, null);
         } else if (!ThingStatus.ONLINE.equals(bridge.getStatus())) {
             setThingStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE, null);
-            getCapabilities().remove(RefreshCapability.class);
-            getCapabilities().remove(ParentUpdateCapability.class);
+            getCapabilities().getParentUpdate().ifPresent(Capability::dispose);
+            getCapabilities().getRefresh().ifPresent(Capability::dispose);
         } else {
             setThingStatus(ThingStatus.UNKNOWN, ThingStatusDetail.NONE, null);
-            if (ModuleType.ACCOUNT.equals(getModuleType().getBridge())) {
-                NAThingConfiguration config = getThing().getConfiguration().as(NAThingConfiguration.class);
-                getCapabilities().put(new RefreshCapability(this, config.refreshInterval));
-            }
-            getCapabilities().put(new ParentUpdateCapability(this));
+            getCapabilities().getParentUpdate().ifPresentOrElse(Capability::initialize, () -> {
+                int interval = getThingConfigAs(NAThingConfiguration.class).refreshInterval;
```

Comment:
It seems to work, but I don't fully understand. This value is defaulted to -1 in code (600 in XML), and in this case `setInterval` will throw `IllegalArgumentException`?

Reply:
Correct, this is a leftover of previous working mode, where -1 was used to trigger  auto-refreshInterval discovery. Changed